mozilla/fx-private-relay
# MPP-2671 Update Relay Email Template

This PR addresses MPP-2671 & MPP-2757 

# New feature description

This updates the email template wrapper for Relay. 

Figma: https://www.figma.com/file/XHGpLK3ARvZDlGu3b8669D/MPPUX-491---Email-Banners?node-id=59%3A124&t=IU7ZxgW71e5gBY3e-0 

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/3924990/218969863-76902913-83d5-4bd0-aa0e-7a7ab83a3d92.png)

![image](https://user-images.githubusercontent.com/3924990/218968709-430a55a8-2a24-469f-ae23-b43d09f68372.png)

# How to test

To test, send an email to one of your masks and check the email received. It should be wrapped using this new template. There are states for both free and premium users. Currently deployed on dev. 

# Checklist (Definition of Done)
